### DISCUSSION TITLE
Okta Enterprise SCIM Provisioning Error - "Error authenticating - No results for user returned"

### DISCUSSION THREAD

**Discussion by alexburythg** [community/community#129719] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am trying to setup SCIM provisioning via Okta, however I'm having an issue as stated in the title, the SAML is setup but the provisioning chucks back an error. The error is "No results for user returned"

![image](https://github.com/community/community/assets/168753397/9f695753-60ae-4d7c-a575-14827bdce822)

Has anyone seen this before and knows the solution?

Thanks,
Alex

**Comment by moemoeq** [comment]
i've also have same issue and got some answer from GH Support team. i hope this answer be helps
--
The reason for the error message Error authenticating: No results for users returned in this instance is because SCIM provisioning at the Enterprise account level with Okta [isn't supported without the EMU feature](https://docs.github.com/enterprise-cloud@latest/admin/identity-and-access-management/using-saml-for-enterprise-iam/configuring-saml-single-sign-on-for-your-enterprise#about-saml-sso).
 
Other customers have requested support for enterprise-level SCIM without EMU and we've added {Redacted}. to the internal issue tracking this. We can't say when or if it will be released but we'd recommend that you keep an eye on our [Changelog](https://github.blog/changelog/) as well as the [GitHub Blog](https://github.blog/) which will reference this feature should it become available at some point in the future.
 
At this time the options available to you are:
[Enable SAML SSO and SCIM only at the individual organization level](https://docs.github.com/enterprise-cloud@latest/organizations/managing-saml-single-sign-on-for-your-organization/configuring-saml-single-sign-on-and-scim-using-okta) (which will require you to [disable it at the enterprise level](https://docs.github.com/enterprise-cloud@latest/admin/managing-iam/using-saml-for-enterprise-iam/disabling-saml-single-sign-on-for-your-enterprise) first).
Use [Enterprise Managed Users](https://docs.github.com/enterprise-cloud@latest/admin/identity-and-access-management/using-enterprise-managed-users-for-iam/about-enterprise-managed-users#getting-started-with-enterprise-managed-users) which [supports enterprise level SCIM](https://docs.github.com/enterprise-cloud@latest/admin/identity-and-access-management/using-enterprise-managed-users-for-iam/configuring-scim-provisioning-for-enterprise-managed-users-with-okta).
 
Note that SCIM without EMU does not create accounts, just invite users to join your organization. Those invitations can only be accepted by a [personal GitHub account](https://docs.github.com/enterprise-cloud@latest/get-started/learning-about-github/types-of-github-accounts#personal-accounts) which has [that address verified on it](https://docs.github.com/enterprise-cloud@latest/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/verifying-your-email-address).
 
 
Regarding the [Enterprise Managed Users (EMU)](https://github.blog/2021-09-30-enterprise-managed-users-generally-available-github-enterprise-cloud/) feature, this provides centralised user management — including provisioning — at the enterprise level. It also provides users with a true SSO experience, which means they would only be required to log in to GitHub using their SAML IdP credentials. An existing enterprise cannot be enabled with EMUs, rather a new EMU-enabled enterprise would need to be created, and your existing assets can be migrated into it. Note that though there are additional abilities, [there are also restrictions on EMU users](https://docs.github.com/enterprise-cloud@latest/admin/identity-and-access-management/understanding-iam-for-enterprises/abilities-and-restrictions-of-managed-user-accounts).
